kurl_util/cmd/pvmigrate/main.go

```diff
@@ -1,15 +1,133 @@
 package main
 
 import (
+	"context"
+	"flag"
 	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
 
+	clusterspace "github.com/replicatedhq/kurl/pkg/cluster/space"
 	"github.com/replicatedhq/kurl/pkg/version"
 	"github.com/replicatedhq/pvmigrate/pkg/migrate"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
 func main() {
+	ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
```

Comment:
Why are we not creating a subcommand for this one? (i.e https://github.com/replicatedhq/kURL/blob/main/pkg/cli/commands.go#L5)

Reply:
Have we not the kurl CLI? How this main will be called?
How/where it will be used?

Reply:
It is called by some of our add-ons like [here](https://github.com/replicatedhq/kURL/blob/aea79861716d66787f0b31670f1fc74a7ee16d1f/scripts/common/rook.sh#L144-L154). This is a wrap of the this [project](https://github.com/replicatedhq/pvmigrate). We just execute a couple of checks prior to calling the `pvmigrate`.

Reply:
I see I think we never would like that someone called it directly.
So, I think we can close this one.